lstchain/image/muon/muon_analysis.py

```diff
@@ -80,33 +70,39 @@ def fit_muon(x, y, image, geom, tailcuts):
     ---------
     muonringparam: ``
     clean_mask:    `np.ndarray` mask after cleaning
-    dist:          `np.ndarray` distance of every pixel 
+    dist:          `np.ndarray` distance of every pixel
                     to the center of the muon ring
     image_clean:   `np.ndarray` image after cleaning
     """
 
-    muonring = ChaudhuriKunduRingFitter(None)
-    clean_mask = tailcuts_clean(geom, image, picture_thresh=tailcuts[0],
-                                    boundary_thresh=tailcuts[1])
+    fitter = ChaudhuriKunduRingFitter()
+    clean_mask = tailcuts_clean(
+        geom, image,
+        picture_thresh=tailcuts[0],
+        boundary_thresh=tailcuts[1],
+    )
     image_clean = image * clean_mask
-    muonringparam = muonring.fit(x, y, image_clean)
+    ring = fitter.fit(x, y, image_clean)
```

Comment:
why `muonringparam ` -> `ring` here and keeping it down?

Reply:
I only edited this function for now. It's a local variable.

Reply:
ah, sorry, forgot one. Actually buggy know